blog/_posts/2020-11-06-hybrid-cloud-with-inlets.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: Hybrid Cloud with inlets
```

Comment:
I think we may do a series, can you suggest a title that's a bit not descriptive?